src/sprites/rendered-target.js

```diff
@@ -170,6 +170,7 @@ RenderedTarget.prototype.setXY = function (x, y) {
     this.x = x;
     this.y = y;
     if (this.renderer) {
+        this.renderer.fencePositionOfDrawable(this.drawableID, this);
```

Comment:
If they can be avoided, I stray away from functions that mutate their arguments both because they are easier to test, and because it's more obvious how they work when you come across them.  For instance here it's possibly that more than `x` and `y` are changing, but you would have to go check the renderer to find out.

Is there a performance reason you chose not to pass in `[x, y]` and have this method return `[newX, newY]`?